test/defaultSnippets.test.ts

```diff
@@ -28,7 +28,14 @@ describe('Default Snippet Tests', () => {
   });
 
   describe('Snippet Tests', function () {
-    function parseSetup(content: string, position: number): Promise<CompletionList> {
+    function parseSetup(content: string, position?: number): Promise<CompletionList> {
+      if (typeof position === 'undefined') {
+        position = content.indexOf('|:|');
+        content = content.replace('|:|', '');
+        if (content.length) position--;
+      }
+      // console.log('position:', position, '>' + content.substring(position) + '<');
```

Comment:
This is how I checked that the `position` of the old and new tests was exactly the same. For example, if the original test called `parseSetup(content, 14)` then I made sure this log output `14` when running the new test.